using-kubecost/navigating-the-kubecost-ui/savings/container-request-right-sizing-recommendations.md

```diff
@@ -0,0 +1,55 @@
+# Container Request Right-Sizing Recommendations
+
+{% hint style="warning" %}
+This feature is in beta. Please read the documentation carefully.
+{% endhint %}
+
+Kubecost can automatically implement its [recommendations](/apis/apis-overview/api-request-right-sizing-v2.md) for container [resource requests](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits) if you have the [Cluster Controller](/install-and-configure/advanced-configuration/controller/cluster-controller.md) component enabled. Using container request right-sizing (RRS) allows you to instantly optimize resource allocation across your entire cluster. You can easily eliminate resource over-allocation in your cluster, which paves the way for vast savings via cluster right-sizing and other optimizations.
+
+## Prerequisites
+
+To receive container right-sizing recommendations, you must first:
+
+* Have a GKE/EKS/AWS Kops cluster
+
+To adopt container right-sizing recommendations, you must:
+
+* Have a GKE/EKS/AWS Kops cluster
```

Comment:
Container request sizing (both recommendations and adoption via 1-click or continuous) has no limitations, it works in all Kubernetes clusters.